experimental/devices/mfrc522/mfrc522.go

```diff
@@ -176,66 +181,188 @@ func (r *Dev) Halt() error {
 //
 //	timeout the duration to wait for IRQ strobe.
 func (r *Dev) SetOperationTimeout(timeout time.Duration) {
+	r.oMu.Lock()
+	defer r.oMu.Unlock()
 	r.operationTimeout = timeout
 }
 
-// Init initializes the RFID chip.
-func (r *Dev) Init() error {
-	if err := r.Reset(); err != nil {
-		return err
+// SetAntennaGain configures antenna signal strength.
+//
+//	gain - signal strength from 0 to 7.
+func (r *Dev) SetAntennaGain(gain int) {
+	r.aMu.Lock()
+	defer r.aMu.Unlock()
+
+	if 0 <= gain && gain <= 7 {
+		r.antennaGain = gain
 	}
-	if err := r.writeCommandSequence(sequenceCommands.init); err != nil {
-		return err
+}
+
+// ReadCard reads the card sector/block.
+//
+// 	auth - the authentication mode.
```

Comment:
I don't think this will come up nice in godoc, you can look locally with:
`godoc -http=localhost:6060`
then browse to http://localhost:6060/pkg/periph.io/x/periph/experimental/devices/mfrc522/